src/subcommands/api.ts

```diff
@@ -0,0 +1,75 @@
+import { Args } from "../args.ts";
+import { API } from "../utils/mod.ts";
+import TokenProvisioner from "../utils/access_token.ts";
+import { error } from "../error.ts";
+import { wait } from "../utils/spinner.ts";
+
+const help = `Perform API calls to any endpoint of the Deploy API (ALPHA)
+
+EXAMPLES:
+
+Get the details of an organization:
+
+    deployctl api organizations/04f19625-35d3-4c05-857e-bcaa3b0af374
+
+Create a project in an organization:
+
+    deployctl --method=POST --body='{"name": "my-project"}' organizations/04f19625-35d3-4c05-857e-bcaa3b0af374/projects
+
+You can find the specification of the API in https://apidocs.deno.com
+
+USAGE:
+    deployctl api [OPTIONS] <ENDPOINT>
+
+OPTIONS:
+    -h, --help                 Prints this help information
+        --method=<HTTP-METHOD> HTTP method to use (defaults to GET)
+        --body=<JSON>          Body of the request. The provided string is sent as is to the API
+        --token=<TOKEN>        The API token to use (defaults to auto-provisioned token)
+`;
```

Comment:
Q: is it intentional to not mention `api-version` option here? (I suppose so, because most likely using versions other than v1 is not something users want to do)

Reply:
Correct, I don't want to give the false idea that v0 is intended for public use.